packages/cli-utils/src/commands/doctor/runner.ts

```diff
@@ -94,21 +95,29 @@ export async function* run(): AsyncIterable<ComposeInput> {
   yield logger.runningTask(Messages.CHECK_DEPENDENCIES);
   await delay();
 
-  const gqlspVersion = deps.find((x) => x[0] === '@0no-co/graphqlsp');
-  if (!gqlspVersion) {
-    yield logger.failedTask(Messages.CHECK_DEPENDENCIES);
-    throw logger.errorMessage(
-      `A version of ${logger.code('@0no-co/graphqlsp')} was not found in your dependencies.\n` +
-        logger.hint(`Is ${logger.code('@0no-co/graphqlsp')} installed?`)
-    );
-  } else if (!semiverComply(gqlspVersion[1], MINIMUM_VERSIONS.lsp)) {
-    yield logger.failedTask(Messages.CHECK_DEPENDENCIES);
-    throw logger.errorMessage(
-      `The version of ${logger.code('@0no-co/graphqlsp')} in your dependencies is out of date.\n` +
-        logger.hint(
-          `${logger.code('gql.tada')} requires at least ${logger.bold(MINIMUM_VERSIONS.lsp)}`
-        )
-    );
+  const supportsEmbeddedLsp = semiverComply(
+    typeScriptVersion[1],
```

Comment:
Since TS is now a peer dependency, we should be able to read this from `require('typescript').version`. Probably have to try-catch it with a new error, but that should also just give us the raw version.

Reply:
Can do that in a follow up